service/provider/src/main/java/gov/va/vro/service/provider/mas/service/mapper/MasCollectionAnnotsResults.java

```diff
@@ -154,6 +159,11 @@ private static AbdCondition createCondition(MasAnnotation masAnnotation) {
     } else {
       abdCondition.setOnsetDate("");
```

Comment:
To be consistent with other dates, should the literal "Z" be removed from the PartialDate?

Reply:
Yes